_sass/base/_highlight.scss

```diff
@@ -88,6 +88,10 @@ table.rouge-table {
 
 .highlight pre {
   width: 100%
+
```

Comment:
Since it's a `scss` file I believe we'd need a `;` to avoid a syntax warning.

```scss
.highlight pre {
  width: 100%;

  ::selection {
    background-color: $base02
  }
}
```

Reply:
Saas accepts indentation although it's a good idea to use put a semi-colon as 99% of the code is using it.